tests/static/all_question_types.xml

```diff
@@ -85,6 +85,7 @@
 					</text>
 					<text id="question1-help">
 						<value>question1 en help</value>
+						<value form="markdown">question1 en help</value>
```

Comment:
There is not - we opted against that design

On Thu, Apr 16, 2015 at 6:13 PM, Daniel Miller <notifications@github.com>
wrote:

> In tests/static/all_question_types.xml
> <https://github.com/dimagi/Vellum/pull/372#discussion_r28554950>:
>
> > @@ -85,6 +85,7 @@
> >  					</text>
> >  					<text id="question1-help">
> >  						<value>question1 en help</value>
> > +						<value form="markdown">question1 en help</value>
>
> Hmm, so we're duplicating the <value> content in <value form="markdown">
> as a flag to say treat this text as markdown. That design seems less than
> ideal. Is there a way for the phone to access the flag that says "treat all
> help text as markdown for this form"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/Vellum/pull/372/files#r28554950>.
>